examples/getting-started-app/README.md

```diff
@@ -17,17 +17,22 @@ Features:
 
 ## Setting up the environment
 
+Для начала разработки на `cartridge` нужно установить несколько утилит:
+
+* `git` - version control system (see details [here](https://git-scm.com/))
+* `npm` - package manager for `node.js` (see details [here](https://www.npmjs.com/))
+* `gcc` - compiler system (see details [here](https://gcc.gnu.org/))
+* `cmake` version no lower than 2.8
+* `tarantool-devel` - development package for `tarantool`
+* `unzip` utility
+
```

Comment:
Before you start developing with `cartridge`, you need to install several utilities:
	
	* `git` &mdash; version control system (see details [here](https://git-scm.com/))
	* `npm` &mdash; package manager for `node.js` (see details [here](https://www.npmjs.com/))
	* `cmake` version 2.8 or higher
	* `tarantool-devel` &mdash; developer package for `tarantool`
	* `gcc` &mdash; `C` compiler (see details [here](https://gcc.gnu.org/))
	* `unzip`